### PR TITLE
added attribute value to image

### LINKED ISSUE #19334: Ensure no missing `alt` attributes
### MDN URL

https://developer.mozilla.org/en-us/learn/css/css_layout/flexbox/

### What specific section or headline is this issue about?

All of MDN content

### What information was incorrect, unhelpful, or incomplete?

333 `alt` attributes are missing. 29 have alt attributes that are the image file name. Untold others have unhelpful alt attributes.

### What did you expect to see?

Every image in MDNs content should have a useful alt attribute (other than tutorials explaining bad practices, maybe)>

### Do you have any supporting links, references, or citations?

List  of missing alt attributes: https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0

### Do you have anything more you want to share?

When writing an alt attribute, think about how you would describe the image to someone who could read the page's content but couldn't load any images. What information does the image convey that is important and relevant to the user?

For example, if there are green boxes with a check mark for good and a red box with an X for bad, the user doesn't need to know the color of the box or the check or X. They do need to know what it is and if it's good or bad.

If there is a screenshot, explain the reason that image was used, not the contents of the screenshot. Don't provide information they don't need or already know. For example, if you're on a page about changing settings on Duck Duck go, if you have a screenshot of duck-duck-go search result, you don't need to explain what the search term is, how many results there are, how long the search took, etc. as that is not the point of the image. Since you're showing how to change settings in duck-duck-go, the alt might be "The settings icon is in the navigation bar below the search field." We didn't include "screenshot" or "duck duck go" as the user doesn't need to know it's a screenshot and already knows it's duck duck go as they are on a page explaining changing duck-duck-go settings.

### PR DESCRIPTION
**Pull request by jhhornn**
#### Related issues
🔨 Fixes #19334  

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/games/techniques/tilemaps/square_tilemaps_implementation_colon__static_maps/index.md [hunk 1]
```diff
@@ -108,4 +108,4 @@ for (let c = 0; c < map.cols; c++) {
 
 Our static tilemap implementation demo pulls the above code together to show what an implementation of this map looks like. You can see a [live demo](https://mozdevs.github.io/gamedev-js-tiles/square/no-scroll.html) and grab the [full source code](https://github.com/mozdevs/gamedev-js-tiles).
 
-[![](no-scroll.png)](https://mozdevs.github.io/gamedev-js-tiles/square/no-scroll.html)
+[![a static aerial view of a tile of field with tress,grass and ground](no-scroll.png)](https://mozdevs.github.io/gamedev-js-tiles/square/no-scroll.html)
```
Comment (estelle) [inline files/en-us/games/techniques/tilemaps/square_tilemaps_implementation_colon__static_maps/index.md:111]:
```suggestion
[![Aerial view of a field with trees, grass, and ground made from repeated sections of the tilemap.](no-scroll.png)](https://mozdevs.github.io/gamedev-js-tiles/square/no-scroll.html)
```

**Review by estelle — APPROVED** [review]
Thank you!

**Comment by jhhornn** [comment]
You're welcome.
I'm happy my first time contributing to open source, my PR got merged.
My system is about to die. I'd do more later today

**Comment by estelle** [comment]
Thanks for contributing and congratulations!

Please don't add the issue number to the 'development' dropdown for this issue. It reads "Successfully merging this pull request may close these issues." This is usually a good thing, but this issue is unique in that it will have a lot of PRs, so we don't want to close it prematurely. thanks.

**Pull request merged**